OpenTibia/Assets/Scripts/Modules/ToggleButtons/ToggleButtonsWidget.cs

```diff
@@ -161,7 +161,27 @@ public class ToggleButtonsWidget : UI.Legacy.SidebarWidget
         }
 
         public void OnLogoutButtonClick() {
+            var gameManager = OpenTibiaUnity.GameManager;
+
+            if (gameManager.IsGameRunning)
+            {
+                void OnLogoutPopupYesClick()
+                {
+                    if (gameManager.IsGameRunning)
+                    {
+                        var protocolGame = OpenTibiaUnity.ProtocolGame;
+                        protocolGame.Disconnect(false);
+                    }
+                }
 
+                void OnLogoutPopupNoClick()
+                {
```

Comment:
Shouldn't we somhow log it at least?

Reply:
I've based on similar parts for existing code. What kind of logging you would like to have?

Reply:
Not pretty sure how it works with Unity, but it'd be nice to have some kind of 'verbose' / 'debug' version of the client in which we could trace each action taken by the user. Testers would be then to drop verbose logs and attach it to every bug they report. In this way we'd have clear view on what was happening with the app at a given time. Is there anything like that in the client?

Reply:
Yes, there is such mechanism like Debug.Log, .LogWarning, .LogError. But in this case is it worth to log "No" button click from that popup window?

Reply:
Let's then agree on at least put: // Intentionally do nothing here :)

Reply:
I agree :)